azblob/zc_sas_query_params.go

```diff
@@ -40,6 +41,37 @@ func FormatTimesForSASSigning(startTime, expiryTime, snapshotTime time.Time) (st
 
 // SASTimeFormat represents the format of a SAS start or expiry time. Use it when formatting/parsing a time.Time.
 const SASTimeFormat = "2006-01-02T15:04:05Z" //"2017-07-27T00:00:00Z" // ISO 8601
+var SASTimeFormats = []string{"2006-01-02T15:04:05.0000000Z", SASTimeFormat, "2006-01-02T15:04Z", "2006-01-02"} // ISO 8601 formats, please refer to https://docs.microsoft.com/en-us/rest/api/storageservices/constructing-a-service-sas for more details.
+
+// formatSASTimeWithDefaultFormat format time with ISO 8601 in "yyyy-MM-ddTHH:mm:ssZ".
+func formatSASTimeWithDefaultFormat(t *time.Time) string {
+	return formatSASTime(t, SASTimeFormats[1]) // By default, "yyyy-MM-ddTHH:mm:ssZ" is used
```

Comment:
>SASTimeFormats[1] [](start = 25, length = 17)

Since we're already using a constant outside of this list, let's avoid magic numbers by referencing it directly (SASTimeFormat), and perhaps rename it to DefaultSASTimeFormat.

Reply:
Ha!! Great suggestion on referring to the constant directly, wish I had realized this myself. 😇 

I don't want to rename `SASTimeFormat` itself though, since you already convinced me to not break the API and this would be a breaking change (https://godoc.org/github.com/Azure/azure-storage-blob-go/azblob#pkg-constants).

Reply:
Good thought, thanks for the catch.